src/components/typography/Paragraph.tsx

```diff
@@ -0,0 +1,38 @@
+import { Text, StyleSheet } from "react-native";
+
+import { global } from "@/constants/DesignTokens";
+
+type ParagraphProps = { size?: "large" | "medium" | "small" | "extraSmall" };
+
+type TextProps = ParagraphProps & Text["props"];
+
+export default function Paragraph(props: TextProps) {
+  const { style, size = "large", ...otherProps } = props;
+
+  return (
+    <Text style={[styles.paragraph, styles[size], style]} {...otherProps} />
+  );
+}
```

Comment:
You can do that without a new line
```suggestion
export default function Paragraph({ style, size = "large", ...otherProps }: TextProps) {
  return (
    <Text style={[styles.paragraph, styles[size], style]} {...otherProps} />
  );
}
```